File: twoSum.js
function whatFlavors(cost, money) {
    var costMap = new Map();
    for (var index = 0; index < cost.length; index++) {
        var complain = money - cost[index];
        if (costMap.has(complain)) {
            var twoCosts = [costMap.get(complain), index + 1]
                .sort(function (a, b) { return a - b; });
            return twoCosts.join(' ');
        }
        costMap.set(cost[index], index + 1);
    }
}
console.log(whatFlavors([1, 4, 5, 3, 2], 4));

File: twoSum.go
func twoSum(nums []int, target int) []int {
	println(nums, target)
	for index, value := range nums {
		complain := target - value
		for i := index + 1; i < len(nums); i++ {
			if nums[i] == complain {
				return []int{index, i}
			}
		}
	}
	return []int{0, 0}
}